WinUIGallery/IdleSynchronizer.cs

```diff
@@ -13,7 +13,7 @@
 
 namespace AppUIBasics
 {
-    class IdleSynchronizer
+    public class IdleSynchronizer
```

Comment:
Is this related to the unit tests?

Reply:
At one point I was using it from one of the unit tests. I'll leave it as it is.